components/webdriver_server/lib.rs

```diff
@@ -1630,34 +1670,65 @@ impl Handler {
                         y: 0,
                         ..Default::default()
                     };
+                    let pointer_move_action = ActionSequence {
```

Comment:
I think we should be dispatching only one action sequence whose `actions` vector contains three actions.